scene/resources/portable_compressed_texture.cpp

```diff
@@ -99,6 +114,7 @@ void PortableCompressedTexture2D::_set_data(const Vector<uint8_t> &p_data) {
 	}
 
 	image_stored = true;
+	size_override = size;
```

Comment:
Good catch. I'm surprised it worked without this before: size_override would have been initialized to zero.